website/learn/glossary/glossary/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: learn
+title: Pactus Glossary
+sidebar: Pactus Glossary
+---
+
+<!-- TODO: find a name for pactus merkel root model and add it here (?) -->
+
+# A
+
+## Account
```

Comment:
```suggestion
### Account
```

Reply:
For glossary we should have a YML file and the compile the final HTML page based on YML file

Reply:
How we can do that?

Reply:
Same as we did in FAQ page. Check that page